CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+* Changed `compas_rhino.uninstall` to also remove broken symlinks if no specific packages are provided for installation.
+* Changed `compas_rhino.install` to also remove broken symlinks if no specific packages are provided for installation.
```

Comment:
It looks like broken symlinks are removed whether or not packages are specified on install.